_sass/index.scss

```diff
@@ -1,17 +1,36 @@
-.hero-section {
-  font-family: 'Helvetica Neue';
+body {
+  margin: 0;
+}
 
-  &__headline {
-    font-size: 1.75rem;
-    margin-bottom: -7rem;
-    width: 21rem;
-    position: relative;
-    z-index: 2;
+.header {
+  color: #ffffff;
+  font-family: 'Swift Neue LT Pro';
+  background: #db9e00;
+
+  &__title {
+    font-size: $font_size-h1;
+    font-weight: 400;
+    margin: 0;
+    padding: 3.75rem 0 .5rem 4.375rem;
+  }
+
+  &__subtitle {
+    font-size: $font_size-h2;
+    font-style: italic;
+    font-weight: 400;
+    margin: 0;
+    padding: 0 0 5.5rem 4.375rem;
+  }
+
+  &__metrocommon-word-mark {
+    font-size: 70%;
   }
 
   &__image {
-    width: 100vw;
-    z-index: -2;
+    background: url('/assets/images/slideshow-1.jpg');
+    background-size: cover;
+    height: 38rem;
+    transition: all 1s ease;
```

Comment:
This is a bit nitpicky (and I'm not sure how to resolve it), but the smooth image transitions don't seem to happen in Firefox. The images do still change, and MDN suggests that this style should work in Firefox, so I'm honestly a bit stumped.

Reply:
Given Firefox has a low user base for us and this isn't major functionality I'm going to just leave this one alone.